cmd/kubeadm/app/util/apiclient/idempotency.go

```diff
@@ -339,10 +339,7 @@ func PatchNode(client clientset.Interface, nodeName string, patchFn func(*v1.Nod
 	return lastError
 }
 
-// GetConfigMapWithRetry tries to retrieve a ConfigMap using the given client,
-// retrying if we get an unexpected error.
-//
-// TODO: evaluate if this can be done better. Potentially remove the retry if feasible.
```

Comment:
this also removes the go comment about the function.

removing the TODO seems OK to me, given it has stayed for a while and we did not change it.
related to https://github.com/kubernetes/kubeadm/issues/1606

Reply:
/easycla

Reply:
Yeah Only That one comment "TODO" needs to be removed, not the go comment too, it happened mistakenly.

And I have signed the CLA